2200065/조교의성적매기기.py

```diff
@@ -0,0 +1,15 @@
+import sys
+sys.stdin = open("input3.txt", "r")
+
+for i in range(1,int(input())+1):
+    n,m = map(int,input().split())
+    grade = ['A+','A0','A-','B+','B0','B-','C+','C0','C-','D0']
+    score = {}
+    for j in range(1,n+1):
+        a,b,c = map(int,input().split())
+        lst = [a*0.35,b*0.45,c*0.2]
+        score[j]= sum(lst)
```

Comment:
깔끔한 풀이네요! 👍
```suggestion
        score[j]= a*0.35 + b*0.45 + c*0.2
```
바로 `score[j]`를 갱신해주셔도 괜찮을 것 같습니다 😄